bsp/src/org/jetbrains/bsp/project/importing/BspResolverLogic.scala

```diff
@@ -309,7 +309,7 @@ private[importing] object BspResolverLogic {
 
     val targetDeps = target.getDependencies.asScala.toSeq
 
-    val data = if(tags.contains(TEST))
+    val data = if(tags.contains(TEST) || tags.contains(INTEGRATION_TEST))
```

Comment:
Can't we use here the `target.getCapabilities.getCanTest` ? 
```suggestion
    val data = if(target.getCapabilities.getCanTest)
```